data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractThreshold.java

```diff
@@ -104,4 +104,6 @@ public boolean isSynchronized() {
 
     @Override
     public abstract int hashCode();
+
+    public abstract void setMargin(double margin);
```

Comment:
Maybe we should add a unit argument to this function ? calls would be less ambiguous